draft-gpew-priv-ppm.md

```diff
@@ -981,6 +981,21 @@ fetch certificates: https://datatracker.ietf.org/doc/html/rfc8555#section-7.4.2]
 issue#57). For example, how does a leader respond to a collect request if the
 helper drops out?]
 
+The collector then decrypts each aggregate share `share` using the following
+procedure:
+
+~~~
+context = SetupBaseR(share.enc, sk,
+                     "ppm aggregate share" || task_id || server_role)
+input_share = context.Open(batch_interval, share.payload)
+~~~
+
+where `sk` is the HPKE secret key, `task_id` is `AggregateReq.task_id` and
+`server_role` is the role of the server that send the aggregate share (`0x01`
+for the leader and `0x00` for the helper). `batch_interval` is the `Interval`
```

Comment:
While this works, wouldn't it be more natural to specify `0x00` for the leader and `0x01` for the helper? This would permit a more natural extension to multiple helpers (which would be able to use `0x02`, `0x03`, ... without placing the leader between the first and second helpers) and would cause these role identifier numbers to match up with the ordering of the input shares in a client report, aggregate shares in an collect result, etc.

(I think this may be somewhat beyond the scope of this individual PR, is this an issue that has been discussed? I notice the interop PR specifies a Role message that has not only the leader & helper but also the collector & client.)

Reply:
Yeah I think this would be a good change, but it should go in a different PR as you suggest.

Reply:
I'd be open to back-porting the language around this from PR #179. There we put leader and helper as `0x02` and `0x03` so that we can allow further helpers to be `0x04` and above, and also extend the info string to include the roles of both sender and recipient. I think that change should be fairly non-controversial, and it'll make it easier to eventually land #179 if we chip little bits of it out.

Reply:
Good catch, @tgeoghegan

Reply:
Backporting this from the interop PR when it is merged into main SGTM.